cloud/provider.go

```diff
@@ -90,14 +90,14 @@ func init() {
 		"mqtt":                "Controls the mqtt protocol config of pulsar cluster",
 		"categories": "Controls the audit log categories config of pulsar cluster, supported categories: " +
 			"\"Management\", \"Describe\", \"Produce\", \"Consume\"",
-		"custom":                 "Controls the custom config of pulsar cluster",
-		"http_tls_service_url":   "The service url of the pulsar cluster, use it to management the pulsar cluster",
-		"pulsar_tls_service_url": "The service url of the pulsar cluster, use it to produce and consume message",
-		"kafka_service_url": "If you want to connect to the pulsar cluster using the kafka protocol," +
+		"custom":                  "Controls the custom config of pulsar cluster",
+		"http_tls_service_urls":   "The service url of the pulsar cluster, use it to management the pulsar cluster",
```

Comment:
Should we maybe update the description to indicate it's a list of strings?

Reply:
fixed